shared/chat/inbox/container/index.tsx

```diff
@@ -204,19 +205,18 @@ const Connected = Container.namedConnect(
       inboxNumSmallRows: stateProps.inboxNumSmallRows,
       isLoading: stateProps.isLoading,
       isSearching: stateProps.isSearching,
-      navKey: ownProps.navKey,
+      navKey: getActiveKey(),
       neverLoaded: stateProps.neverLoaded,
       onNewChat: dispatchProps.onNewChat,
       onUntrustedInboxVisible: dispatchProps.onUntrustedInboxVisible,
       rows,
       setInboxNumSmallRows: dispatchProps.setInboxNumSmallRows,
       smallTeamsExpanded: stateProps.smallTeamsExpanded,
-      title: 'Chats',
       toggleSmallTeamsExpanded: dispatchProps.toggleSmallTeamsExpanded,
       unreadIndices,
     }
   },
-  'Inbox'
+  'Chats'
```

Comment:
?? this can break stories

Reply:
I think I had removed this line in order to use `connectDEBUG` and then when I added it back I added the wrong thing